src/services/observation.service.ts

```diff
@@ -565,3 +577,30 @@ export const axGetPlantnetSuggestions = async (imageUrls, organs) => {
     return { success: false, data: [] };
   }
 };
+
+export const axGetGroupByDayCreatedOn = async (userId) => {
+  try {
+    const { data } = await plainHttp.get(
+      `${ENDPOINT.OBSERVATION}/v1/observation/user/${userId}`
+    );
+    return { success: true, data };
+  } catch (e) {
+    return { success: false, data: [] };
+  }
+};
+
+export const axAddDownloadLog = async(source,filterUrl,notes) => {
```

Comment:
Move this download log service  to user service. Also update this to accept a payload which contains all your variables. You can find references in other similar POST Apis.